nlmod/mdims/mgrid.py

```diff
@@ -52,7 +56,7 @@ def xy_to_icell2d(xy, ds):
     return icell2d
 
 
-def modelgrid_from_model_ds(model_ds):
+def modelgrid_from_model_ds(model_ds, rotated=True):
```

Comment:
Do we need the `rotated` argument? Is checking for `angrot`==0.0 not enough?

Reply:
Sometimes you want to have a modelgrid-object in model coordinates. When plotting with nlmod.plot.da() the method get_vertices() is called, which generates a modelgrid object with the default value for the argument 'rotated' set to False. But I think we can code this more elegantly and faster in the future, by skipping the modelgrid object and generating vertices streight from model_ds. Right not get_vertices() can take a while...